pkg/kubectl/cmd/get/humanreadable_flags.go

```diff
@@ -66,7 +66,6 @@ func (f *HumanPrintFlags) ToPrinter(outputFormat string) (printers.ResourcePrint
 		return nil, printers.NoCompatiblePrinterError{Options: f}
 	}
 
-	encoder := scheme.Codecs.LegacyCodec(scheme.Registry.RegisteredGroupVersions()...)
```

Comment:
@liggitt one does has to wonder what exactly you'd be re-encoding.  Might be able to remove the entire raw case.